pkg/resources/database_acceptance_test.go

```diff
@@ -1112,8 +1124,13 @@ func TestAcc_Database_WithReplication(t *testing.T) {
 				),
 			},
 			{
-				ProtoV6ProviderFactories: acc.TestAccProtoV6ProviderFactories,
-				Config:                   databaseStateUpgraderWithReplicationNew(id, secondaryAccountIdentifier),
+				ExternalProviders: map[string]resource.ExternalProvider{
+					"snowflake": {
+						VersionConstraint: "=0.94.1",
+						Source:            "Snowflake-Labs/snowflake",
+					},
+				},
+				Config: databaseStateUpgraderWithReplicationNew(id, secondaryAccountIdentifier),
```

Comment:
Why are these tests pinned to a version?

Reply:
Right now, no. Changed to the latest version.